kaldb/src/main/java/com/slack/kaldb/server/ManagerApiGrpc.java

```diff
@@ -250,6 +276,28 @@ protected static List<SnapshotMetadata> fetchSnapshots(
     return snapshotMetadata;
   }
 
+  /**
+   * Fetches all SnapshotMetadata that match the IDs in snapshotIds
+   *
+   * @return List of SnapshotMetadata that are within specified timeframe and from queried service
+   */
+  protected static List<SnapshotMetadata> fetchSnapshots(
```

Comment:
Small nit - this isn't really "fetching" as much as it is just calculating the value from the provided values.